package.json

```diff
@@ -13,7 +13,7 @@
     "lint:imports": "node scripts/check-imports.js",
     "lint": "npm run lint:typescript && npm run lint:lit",
     "lint:typescript": "eslint \"packages/**/*.ts\"",
-    "lint:lit": "lit-analyzer \"packages/**/*.ts\" --strict --quiet",
+    "lint:lit": "lit-analyzer \"packages/**/*.ts\" --strict",
```

Comment:
Why remove the `--quiet` argument?

Reply:
because it's not quiet internally and fails on build. This will not fail build externally if there is a problem with the linter.

Reply:
Ugh, `--quiet` not setting the exit code is not what I expected 😕